client/src/actions/index.js

```diff
@@ -0,0 +1,140 @@
+import axios from 'axios';
```

Comment:
Nice! I know you showed us your logs but I forgot that you used Redux in here. Good job!